__tests__/errorsHandled.test.js

```diff
@@ -34,6 +33,33 @@ describe("GET /api/:invalid endpoint", ()=>{
     test("Returns 404 error", ()=>{
         return request(app)
         .get('/api/invalid')
-        .expect(404);
+        .expect(404) 
+        .then((res)=>{
+            //console.log(Object.keys(res));
+            expect(res.body.msg).toBe("Invalid endpoint");
+        })
+    })
+})
+
+describe("GET /api/reviews/:review_id", ()=>{
+    test("GET /api/reviews responds with a review object, which has the following keys: review_id, title, review_body, designer, review_img_url, votes, category, owner, create_at", ()=>{
+        return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then((res)=>{
+            console.log(res.body[1]);
+            //expect the review object to have keys:
+            expect(res.body[1]).toEqual(expect.objectContaining({
+                review_id:expect.any(Number),
+                title:expect.any(String),
+                review_body:expect.any(String),
+                designer:expect.any(String),
+                review_img_url:expect.any(String),
+                votes:expect.any(Number),
+                category:expect.any(String),
+                owner:expect.any(String),
+                created_at:expect.any(String),
+            }))
+        })
```

Comment:
you can be more specific here, if you are looking for reviews/1 then we know the review_id should be 1. 

we also know what the title, review_body etc are all exactly going to look like so instead of expecting any string/number, have a look at the test data or the console.log res.body and be more specific, you can just put the values in instead of the expects.